README.md

```diff
@@ -1,29 +1,29 @@
 # Composite Container-based Application Specification
 
-`/NOO-le-kyul/` (n.) a made-up word meaning ["the mother of all atomic particles"](http://simpsons.wikia.com/wiki/Made-up_words). Sounds like "molecule". But different.
+`/NOO-le-kyul/` (n.) a made-up word meaning ["the mother of all atomic particles"](http://simpsons.wikia.com/wiki/Made-up_words). It is spelled like "molecule", but pronounced like "noodle".
 
-**Your installer for containers.** Replace your shell script and deployment instructions with some metadata.
+**Your installer for container-based applications.** Replace your shell script and deployment instructions with some metadata.
 
 **Change runtime parameters for different environments.** No need to edit files before deployment. Users can choose interactive or unattended deployment. Guide web interface users with parameter metadata to validate user input and provide descriptive help.
 
 **Bridge between Enterprise IT and PaaS** With pluggable orchestration providers you can package your application to run on OpenShift, Kubernetes, Docker Compose, Helios, Panamax, Docker Machine, etc. and allow the user to choose the target when deployed.
 
-**Compose applications from a catalog.** No need to re-package common services. Create composite applications by referencing other Nulecule-compliant apps. Adding a well-designed, orchestrated database is simply a reference to another container image.
+**Compose applications from a catalog.** No need to re-package common services. Create composite applications by referencing other Nulecule-compliant apps. For example, adding a well-designed, orchestrated database is simply a reference to another container image.
 
 ## Problem Statement
-Currently there is no standard mechanism to define a composite multi-container application or composite service composed of aggregate pre-defined building blocks spanning multiple hosts and clustered deployments. In addition, the associated metadata and artifact management requires separate processes outside the context of the application itself. 
+Currently there is no standard mechanism to define a composite multi-container application or composite service composed of aggregate pre-defined building blocks spanning multiple hosts and clustered deployments. Additionly, the associated metadata and artifact management currently requires a separate management and storage process that is unrelated to the storage and management process used for the containers that are the application or service.
 
```

Comment:
@goern rather than escalate, let's propose a change.  I believe we will get a faster result that way.

How about a totally new line:

Currently there is no standard way of defining a multi-container application's configuration without distributing instructions and files to the end-user.  Additionally, these files must be managed and distributed via different systems than the containers themselves.